pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -213,5 +212,10 @@ func pipeLogs(logC chan<- Log, errC chan<- error, tlr *taskrun.LogReader) {
 }
 
 func empty(status v1alpha1.PipelineRunStatus) bool {
+
+	if status.Status.Conditions == nil {
```

Comment:
This should be `status/Status == nil` according to me

Reply:
`status.Status` is a struct so it can't be `nil` (i.e. `cannot convert nil to type "knative.dev/pkg/apis/duck/v1beta1".Status`). Am I missing something?

Reply:
Oh sorry, I mean `status.Conditions == nil `

Reply:
Yeah, good catch. I pushed up a change for it. Thanks.